lib/decimal/decimal.h

```diff
@@ -1,16 +1,10 @@
-/* Copyright (c) 2018-2019, NVIDIA CORPORATION.  All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+/*===-- lib/decimal/decimal.h -------------------------------------*- C++ -*-===
+ * 
+ *  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ *  See https://llvm.org/LICENSE.txt for license information.
+ *  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ * 
```

Comment:
Line 3 is 81 characters long so clang-format will reformat the comment and mess it up. I suggest taking out the space after the asterisk in each of these lines.

Reply:
Yes, please verify that clang-format on all modified files is a no-op.

Reply:
It looks like an extra space did get into those three lines after the asterisk. I will make sure it's removed. One space is sufficient, and that will leave it at 80.

I will also run everything through clang-format as suggested.

Reply:
After fixing the two comments mentioned here and running clang-format, there were three files that had changes that were unrelated to the relicensing changes. I plan to push the updated branch _without_ these changes. (This was using clang-format as specified in Pete's document: `/home/sw/thirdparty/llvm/linux86-64/7.1.0-f18/bin/clang-format`.) Please let me know if you object to that.

```diff --git a/lib/evaluate/intrinsics.cc b/lib/evaluate/intrinsics.cc
index a18d65a..97ef72a 100644
--- a/lib/evaluate/intrinsics.cc
+++ b/lib/evaluate/intrinsics.cc
@@ -594,7 +594,8 @@ static const IntrinsicInterface genericIntrinsicFunction[]{
         Rank::scalar},
     {"rank", {{"a", AnyData, Rank::anyOrAssumedRank}}, DefaultInt,
         Rank::scalar},
-    {"real", {{"a", SameComplex, Rank::elemental}}, SameReal},  // 16.9.160(4)(ii)
+    {"real", {{"a", SameComplex, Rank::elemental}},
+        SameReal},  // 16.9.160(4)(ii)
     {"real", {{"a", AnyNumeric, Rank::elementalOrBOZ}, DefaultingKIND},
         KINDReal},
     {"reduce",
diff --git a/lib/parser/type-parser-implementation.h b/lib/parser/type-parser-implementatio
index aa25130..c1611a5 100644
--- a/lib/parser/type-parser-implementation.h
+++ b/lib/parser/type-parser-implementation.h
@@ -33,4 +33,3 @@
   TYPE_PARSER(CONTEXT_PARSER((contextText), (pexpr)))
 
 #endif
-
diff --git a/test/evaluate/fp-testing.cc b/test/evaluate/fp-testing.cc
index d7df7af..98a84de 100644
--- a/test/evaluate/fp-testing.cc
+++ b/test/evaluate/fp-testing.cc
@@ -11,7 +11,7 @@ ScopedHostFloatingPointEnvironment::ScopedHostFloatingPointEnvironment(
 #else
     bool, bool
 #endif
-    ) {
+) {
   errno = 0;
   if (feholdexcept(&originalFenv_) != 0) {
     std::fprintf(stderr, "feholdexcept() failed: %s\n", std::strerror(errno));```

Reply:
Why not also push the formatting fixes?  Somebody else will just have to do it later otherwise.

Reply:
Mostly to have the relicensing change PR only be relicensing changes. Also, because I wasn't confident that I have the environment set up the same as you all do, and that these were spurious.

I would happily do a separate PR for these three changes if you like.